pkg/controller/druid/handler.go

```diff
@@ -328,6 +332,37 @@ func deployDruidCluster(sdk client.Client, m *v1alpha1.Druid) error {
 	return nil
 }
 
+func checkCrashStatus(sdk client.Client, m *v1alpha1.Druid) {
+	podList := podList()
+	if err := sdk.List(context.TODO(), podList); err != nil {
```

Comment:
this would list all pods in namespace where operator is running? instead we would want to list all pods in the user spec's namespace in a given statefulSet .. no?

Reply:
we will call this function inside the sts block plus will wrap this in an if condition such as 

if sts.Podmanagement == "orderReady" {
checkCrashStatys()
}

so should be good

Reply:
even then, I would say we should only ask to list pods that we are concerned about, not every pod ... we should be able to do that by using right namespace and label selectors.

Reply:
sure agreed, can add namespace to []client.list but not sure what  label selector to use  unless a map[string]string exists specifically for filtering MM.

Reply:
I think we already add following labels to all the pods...

```
	labels["app"] = "druid"
	labels["druid_cr"] = clusterName
	labels["nodeSpecUniqueStr"] = nodeSpecUniqueStr
```

above labels combined with namespace should identify all pods for a specific statefulset/deployment operator created.

Reply:
yeah i added makelabels()  :)